server/test/test_diffexp.py

```diff
@@ -105,13 +105,23 @@ def sparse_diffexp(self, apply_col_shift):
             adata = adaptor_anndata.data
 
             sparsename = os.path.join(dirname, "sparse.cxg")
-            write_cxg(adata=adata, container=sparsename, title="sparse", sparse_threshold=11)
+            cxg_group_metadata = create_cxg_group_metadata(
```

Comment:
Do we also need a test case here where `corpora_props` is not None since that is new functionality?

Reply:
@bmccandless - can you respond.  I'm unclear why this parameter would be necessary to test for differential expression.

Reply:
This test is just to verify that the sparse matrix converter works.   
A test case for corpora_props should probably go in the test_cxgtool.py

Reply:
write_cxg is not a public interface, and will never have a None value in props.  Matt and Brian pulled  this out simply for testing.

I will add a test case for the corpora.py exported API.

Reply:
added test_corpora.py